From 4fe169b5cb6b0946348eb6a2dd9092f166cc1d67 Mon Sep 17 00:00:00 2001
From: Zhang Xiaodong <B39117@freescale.com>
Date: Fri, 28 Dec 2012 14:41:32 +0800
Subject: [PATCH 378/505] ENGR00224755 mxc cec test:Modify makefile of
 hdmi-cec test

Modify makefile of hdmi-cec test for build mxc_cec_test.out.

Signed-off-by: Zhang Xiaodong <B39117@freescale.com>
---
 test/mxc_cec_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_cec_test/Makefile b/test/mxc_cec_test/Makefile
index 264b5d1..98128fa 100755
--- a/test/mxc_cec_test/Makefile
+++ b/test/mxc_cec_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for MediaLB driver test app.
 #
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX6Q
+INCLUDE_LIST:= IMX6Q
 
 INC += -I$(LINUXPATH)/include
 CFLAGS += -lpthread -lcec -lrt
-- 
1.8.0

